import-scripts/annotation_suite/standardize_mutation_data.py

```diff
@@ -0,0 +1,1414 @@
+    print("\nLoading data from input directory: %s" % (input_directory))
+    print("\n\tSearching for files with extensions: %s " % (", ".join(extensions)))
+
+    for filename in os.listdir(input_directory):
+        extract_data = False
+        for ext in extensions:
+            if filename.endswith(ext) and "normal" not in filename:
+                extract_data = True
+                break
+        if extract_data:
+            output_filename = os.path.join(output_directory, filename + ".temp")
+            if has_vcf_data and ".vcf" in filename:
+                maf_data = extract_vcf_data_from_file(os.path.join(input_directory, filename), center_name, sequence_source)
+            else:
+                maf_data = extract_maf_data_from_file(os.path.join(input_directory, filename), center_name, sequence_source)
+            write_standardized_mutation_file(maf_data, output_filename)
+
+def is_valid_record(maf_data):
+    if not maf_data["Chromosome"] in VALID_CHROMOSOMES:
+        return False
+    return True
+
+def usage():
+    print("python3 standardize_mutation_data.py --input-directory [path/to/input/directory] --output-directory [path/to/output/directory] --center [default name for center] --sequence-source [WGS | WXS] --extensions [comma-delimited list of extensions]")
+    sys.exit(1)
+
+def main():
+    parser = optparse.OptionParser()
+    parser.add_option("-i", "--input-directory", action = "store", dest = "inputdir", help = "input data directory")
+    parser.add_option("-o", "--output-directory", action = "store", dest = "outputdir", help = "output data directory")
+    parser.add_option("-c", "--center", action = "store", dest = "centername", help = "name of center (standard MAF field = 'Center')")
+    parser.add_option("-s", "--sequence-source", action = "store", dest = "seqsource", help = "Sequencing source (standard MAF field = 'Sequencing_Source'), e.g., WXS or WGS")
+    parser.add_option("-x", "--extensions", action = "store", dest = "extensions", help = "File extensions to look for (e.g., .vcf, .maf), comma-delimited list")
+
+    (options, args) = parser.parse_args()
+    input_directory = options.inputdir
+    center_name = options.centername
+    output_directory = options.outputdir
+    sequence_source = options.seqsource
+    extensions = options.extensions
+
+    if not input_directory or not output_directory or not center_name or not extensions:
+        print("Input directory, output filename, extension(s) to search for, and center name must be provided.")
+        usage()
+    if not sequence_source:
+        sequence_source = ""
+    else:
+        sequence_source = sequence_source.upper()
+    if not center_name:
+        center_name = "."
+
+    has_vcf_data = ("vcf" in extensions)
+    generate_maf_from_input_data(input_directory, output_directory, extensions.split(","), center_name, sequence_source, has_vcf_data)
```

Comment:
do we need to trim the extensions here?  In case the user does `--extensions="vcf, maf, txt    "` ?